can-component.js

```diff
@@ -492,9 +492,24 @@ var Component = Construct.extend(
 				} else if (componentTagData.viewModel) {
 					// Component is being instantiated with a viewModel
 					setupFn = getSetupFunctionForComponentVM(componentTagData.viewModel);
+
+					//!steal-remove-start
+					if (process.env.NODE_ENV !== 'production') {
+						Object.defineProperty(setupFn, "name",{
+							value: "render <"+this.tag+">",
+							configurable: true
+						});
+
+						setupFn = queues.runAsTask(setupFn, function(el, componentTagData) {
+							return ["Rendering", el, "with",el, componentTagData.viewModel];
```

Comment:
In this case, the component is not being rendered. The case that existed already([this one](https://github.com/canjs/can-component/blob/ad8428bde67dd98590cf734c73baead2db0c8e0e/can-component.js#L395)) is called when an element is inserted into a page using stache (like `{{ componentInstance }}`), which is why it is queued as `Rendering ...`.

This case the element is being constructed.

Right now this message says

> Rendering `<a-tag>​</a-tag>`​ with `<a-tag>​</a-tag>`​ {foo: Observation}

I think it should say something like

> Constructing `<a-tag>​</a-tag>`​ with viewModel​ {foo: Observation}